formulas/functions/information.js

```diff
@@ -63,8 +63,11 @@ const InfoFunctions = {
     return typeof value !== "string";
   },
 
-  ISNUMBER: (value) => {
-    value = H.accept(value);
+  ISNUMBER: (inputValue) => {
+    const value = H.accept(inputValue, null, undefined, false);
+    if(Array.isArray(value)){
```

Comment:
Read https://github.com/mkirchner/linked-list-good-taste

Then you'll see why we should do
```
const acceptedValue = H.accept(inputValue, null, undefined, false);
let value;
if(Array.isArray(acceptedValue)) {
  value = acceptedValue;
} else {
  value = [[acceptedValue]];
} 
return value.map((row) => row.map((val) => typeof val === "number"));
```